pootle/apps/pootle_project/forms.py

```diff
@@ -40,4 +46,20 @@ class Meta(object):
             model = TranslationProject
             fields = ('language', 'project')
 
+        def save(self, commit=True):
+            initialize_from_templates = False
+            tp = self.instance
+            if not tp.pk:
+                initialize_from_templates = tp.can_be_inited_from_templates()
+
+            tp = super(TranslationProjectForm, self).save(commit)
+            if initialize_from_templates:
+                queue = get_queue('default')
+                if queue._async:
+                    queue.enqueue(init_tp_from_templates, tp)
+                else:
+                    init_tp_from_templates(tp)
```

Comment:
As far as I can see this doesn't check if the TP is already being initialized from templates.